qutebrowser/config/configtypes.py

```diff
@@ -798,6 +799,17 @@ class File(BaseType):
 
     typestr = 'file'
 
+    def transform(self, value):
+        if not value:
+            return None
+        value = os.path.expanduser(value)
+        if not os.path.isabs(value):
+            if standarddir.config():
```

Comment:
I included the check in validate, updated the corresponding test and added the new test (had to add a mock for os_path.isabs), however when removing the test for the return of ```standarddir.config()``` ```test_transform_base64``` of ```TestUserStyleSheet``` fails.

The test isn't calling validate before transform, but even if it would, it would catch the ValidationError and continue even though standarddir.config() returns None, at which point it will be trigger that same trap in File.transform. I think the easiest solution is to just leave the test for the return of standarddir.config() in File.transform.